packages/ketcher-core/src/application/editor/data/monomers.ket

```diff
@@ -86966,6 +86975,691 @@
+          0
+        ]
+      },
+      {
+        "label": "C",
+        "location": [
+          6.272100475741676,
+          0.5035988271634368,
+          0
+        ]
+      },
+      {
+        "label": "N",
+        "location": [
+          7.015243103308636,
+          -0.16556963017329673,
+          0
+        ]
+      },
+      {
+        "label": "C",
+        "location": [
+          4.944903635394974,
+          -1.7177069852364422,
+          0
+        ]
+      }
+    ],
+    "bonds": [
+      {
+        "type": 1,
+        "atoms": [
+          0,
+          1
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          1,
+          2
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          2,
+          3
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          0,
+          4
+        ]
+      },
+      {
+        "type": 2,
+        "atoms": [
+          2,
+          5
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          0,
+          6
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          1,
+          7
+        ],
+        "stereo": 1
+      },
+      {
+        "type": 1,
+        "atoms": [
+          7,
+          8
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          8,
+          9
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          9,
+          10
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          10,
+          11
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          11,
+          12
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          12,
+          13
+        ]
+      },
+      {
+        "type": 2,
+        "atoms": [
+          12,
+          14
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          13,
+          18
+        ]
+      },
+      {
+        "type": 2,
+        "atoms": [
+          18,
+          17
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          17,
+          16
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          16,
+          15
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          15,
+          19
+        ]
+      },
+      {
+        "type": 1,
+        "atoms": [
+          13,
+          15
+        ],
+        "stereo": 1
+      }
+    ],
+    "class": "AminoAcid",
+    "classHELM": "PEPTIDE",
+    "id": "meO___N-Methyl-Pyrrolysine",
+    "fullName": "N-Methyl-Pyrrolysine",
+    "alias": "meO",
+    "attachmentPoints": [
+      {
+        "attachmentAtom": 1,
+        "leavingGroup": {
+          "atoms": [
+            3
+          ]
+        },
+        "type": "left"
+      },
+      {
+        "attachmentAtom": 2,
+        "leavingGroup": {
+          "atoms": [
+            5
```

Comment:
Leaving groups for this monomer are still not defined correctly, please fix this:
R1=H (0-4), R2=OH (2-3)